contributed_definitions/nyaml/NXopt_assembly.yaml

```diff
@@ -0,0 +1,47 @@
+category: base 
+doc: |
+  Contains datastructures of an experimental optical setup (i.e., multiple transfermatrix tables).
+  These datastructures are used to relate physical properties of two beams
+
+  (NXopt_beam) which have one common optical element (NXopt_element) (one 
+  specific transfermatrix).
+  One of these beams in an input beam and the other one is an output beam.
+
+  The data describes the change of beam properties, e.g. the intensity of a beam is reduced because the transmission coefficient of the opt element is lower than 1.
+symbols:
+  doc: |
+    Variables used throughout the document, e.g. dimensions or parameters.
+  N_variables: |
+    Length of the array associated to the data type.
+opt_assembly(NXobject): # Which properties of NXdata are required?

```

Comment:
Should be called NXopt_assembly, all classes have to start with NX.

Reply:
actually, renamed to NXbeam_transfer_matrix_table and merged with NXtransfer_matrix_tabel